### DISCUSSION TITLE
Welcome to udm-patches Discussions!

### DISCUSSION THREAD

**Discussion by cpriest** [cpriest/udm-patches#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ajappe** [comment]
what generates the openvpn-N.conf?? bit lost with that

**Comment by cpriest** [comment]
That's what you provide to configure the VPN connection, typically this would be the same as what's provided by your VPN server administrators like from an OpenVPN server or some such.

I believe I have a trimmed/anonymized example openvpn config in the repo as a sample.
